openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/SearchIndexApplication.spec.ts

```diff
@@ -100,11 +96,7 @@ test('Search Index Application', async ({ page }) => {
     await page.click('.ant-modal-body [data-testid="deploy-button"]');
     await deployResponse;
 
-    await expect(page.getByRole('alert').first()).toHaveText(
-      /Schedule saved successfully/
-    );
-
-    await page.getByLabel('close').first().click();
+    await toastNotification(page, 'Schedule saved successfully');
```

Comment:
this one also regex

Reply:
this will work as in utils we are checking with `toHaveText` which will check the entire string!